yarn-project/simulator/src/avm/journal/journal.test.ts

```diff
@@ -129,7 +128,7 @@ describe('journal', () => {
       const utxo = new Fr(2);
```

Comment:
Do you know why this is called an UTXO? Where it was called a msgHash in other tests in this pr?